da/celestia/proto/celestia.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+package celestia;
+
+option go_package = "github.com/celestiaorg/optimint/da/celestia/pb";
+import "tendermint/types/types.proto";
+
+service DataRetriever {
+	rpc RetrieveShares (Request) returns (Shares) {}
+	rpc RetrieveStateRelevantMessages (Request) returns (StateRevelantMessages) {}
+	rpc RetrieveMessages (Request) returns (tendermint.types.Messages) {}
+}
+
+message DataAvailabilityHeader {
+	repeated bytes row_roots = 1;
+	repeated bytes column_roots = 2;
+}
+
+message Request {
```

Comment:
`Request` is too unspecific IMO. It should be `NamespaceRequest` or `NamespaceDataRequest` or sth alike.